articles/Choosing-between-Django-Flask-and-FastAPI/index.md

```diff
@@ -0,0 +1,79 @@
+2. Django ORM: Django models can save themselves and transactions of the default because it was created before SQLAlchemy came into existence.
+
+##### Django: Use cases
+You can use Django to tackle projects of any size and capacity, from simple sites to high-performance sites. It can be used for high load booking engines or shopping platforms, Documentary management systems, Built-in custom CRM systems for internal data, and even IOS and Andriod applications that support web-based applications. Some developer features that can be efficiently implemented with Django is Admin dashboards, Photo-based verification, Emailing systems for sending user notifications, etc.
+
+#### Flask
+It is a micro web framework written in Python. It is built to make building web apps with Python easy and fast and scales up very complex applications. It was more like an April fool’s prank by the creator Armin Ronacher on the 1st of April, where he wrapped Werkzeug and Jinja, which were both created by him. Surprisingly, it was appreciated, which made it to be taken seriously, and more work was done on it.
+
+##### Pros
+1. Flexible and Easy: Most of the parts of Flask have the possibility of changing, which is very unlikely for some other web frameworks.
+2. Testing: Flask allows unit testing and would enable you to transit to a web framework by tweaking some extensions because of its built-in-development server, integrated support, etc.
+3. Easy to Understand: Flask is pretty much very beginner-friendly because of its simplicity, giving room to learn and understand it better. It also enables devs to create apps effortlessly quickly. 
+
+#### Cons
+1. Modules: Flask uses Modules, a third-party involvement prone to cause security breaches. The modules are now like a third-party between the framework and the developer.
+2. Scale: Flask has a singular source that implies that it will handle every request in turns, one after the other, so regardless of how many multiple requests it still takes them in turns, it takes more time.
+
+##### Flask: Use cases
+Flask can be used for commercial projects. It can help you get started quickly but doesn’t work well for resembling a real load. You can easily implement projects with Flask: E-commerce systems, Facebook/ Twitter bot, An online social network, static sites, etc. Still, it’s not advisable to be used for high-load enterprise software.
+
+#### FastAPI  
+Fast API is a modern open-source fast and highly performant Python web framework for building Web APIs and is based on Python 3.6+ standard type hints. 
+
+##### Pros
+1. Data validation: Fast API validates the developer’s data type even in deeply nested JSON requests.
+2. Standards: Fast API is built on standards like JSON Schema, OAuth 2.0 and OpenAPI
```

Comment:
Why do these standards matter? Don't give one-line explanations for the pros and cons. Sell the idea that you are trying to make with facts, presented concisely. As a reader, my attention span is limited, and the more unknown variables I see, the more I drift. You talk about standards such as OAuth. Since this is an article intended for beginners, you need to explain every detail of it.